patroni/ctl.py

```diff
@@ -646,3 +646,59 @@ def configure(config_file, dcs, namespace):
     config['dcs_api'] = str(dcs)
     config['namespace'] = str(namespace)
     store_config(config, config_file)
+
+
+def touch_member(config, dcs):
+    ''' Rip-off of the ha.touch_member without inter-class dependencies '''
+    p = Postgresql(config['postgresql'])
+    p.set_state('running')
+    p.set_role('master')
+
+    api = RestApiServer(None, config['restapi'])
```

Comment:
It will start listening on the REST API port, what means it would be not possible to run `scaffold` if there is already Patroni running on the same machine with the same config.